themes/scss/map/dashboard-info.scss

```diff
@@ -119,9 +117,9 @@
 .CDB-Dashboard-infoFooterTxt {
   display: inline-block;
   color: #FFF;
-  font-size: $sFontSize-larger;
+  font-size: $sFontSize-large;
   font-weight: $sFontWeight-lighter;
-  line-height: $sLineHeight-larger;
+  line-height: $sLineHeight-large;
```

Comment:
Name of variable `sLineHeight-large` should be written in all lowercase letters with hyphens instead of underscores